src/framework/components/anim/component-layer.js

```diff
@@ -229,7 +229,13 @@ class AnimComponentLayer {
     }
 
     set activeStateCurrentTime(time) {
+        const layerPlaying = this._controller.playing;
```

Comment:
you access this._controller many times here .. do
`const controller = this._controller;`